wagtail_picture/templates/responsive_picture.html

```diff
@@ -0,0 +1,20 @@
+{% load wagtailimages_tags %}
+{# width- resize rules shouldn’t be hard-coded, but there is no way for us to provide it via Django Templates’ include, so we have to pretend #}
```

Comment:
so i'm presuming a future task is for us to allow `resize_rule` to be passed via the `include` and the accepted by the `{% image %}` tag?

Reply:
Yep, which is unfortunately impossible to do with `{% include %}` – hence why I think new template tags will be the only option